client/src/components/files/FileListDisplay.vue

```diff
@@ -120,10 +120,13 @@ const someSelected = computed(() => {
 });
 
 onMounted(async () => {
-  containerRef.value.addEventListener('contextmenu', (event: Event) => {
-    event.preventDefault();
-    emits('globalMenuClick', event);
-  });
+  // Bit of a nightmare in web mode to inspect
+  if (!isWeb()) {
```

Comment:
Hmm this is just temporary right?
Like I get that for developers it is useful to be able to inspect, but we still want our contextual menu for future web users.